Test/error_test.py

```diff
@@ -8,121 +8,121 @@
+pycm.pycm_obj.pycmMatrixError: Invalid input confusion matrix format.
 >>> cm_4=ConfusionMatrix(matrix={1: {1: 2, "1": 2}, "1": {1: 2, "1": 3}})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmMatrixError: Type of the input matrix classes is assumed  be the same
+pycm.pycm_obj.pycmMatrixError: All input matrix classes must be of the same type.
 >>> cm_5=ConfusionMatrix(matrix={1: {1: 2}})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmMatrixError: Number of the classes is lower than 2
+pycm.pycm_obj.pycmMatrixError: The number of classes must be at least 2.
 >>> cm = ConfusionMatrix([1, 2, 3, 4], [1, 2, 3, 4], classes=[1])
 Traceback (most recent call last):
         ...
-pycm.pycm_error.pycmMatrixError: Number of the classes is lower than 2
+pycm.pycm_error.pycmMatrixError: The number of classes must be at least 2.
 >>> cm = ConfusionMatrix([1, 1, 1, 1], [1, 2, 1, 1], classes=[])
 Traceback (most recent call last):
         ...
-pycm.pycm_error.pycmMatrixError: Number of the classes is lower than 2
+pycm.pycm_error.pycmMatrixError: The number of classes must be at least 2.
 >>> y_actu = [2, 0, 2, 2, 0, 1, 1, 2, 2, 0, 1, 2]
 >>> y_pred = [0, 0, 2, 1, 0, 2, 1, 0, 2, 0, 2, 2]
 >>> cm = ConfusionMatrix(y_actu, y_pred)
 >>> cm.distance(metric = 2)
 Traceback (most recent call last):
         ...
-pycm.pycm_error.pycmMatrixError: The metric type must be DistanceType
+pycm.pycm_error.pycmMatrixError: The metric type must be DistanceType.
 >>> cm.relabel([1, 2, 3])
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmMatrixError: Mapping format error
+pycm.pycm_obj.pycmMatrixError: Invalid mapping format.
 >>> cm.relabel({1: "L1", 2: "L2"})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmMatrixError: Mapping class names error
+pycm.pycm_obj.pycmMatrixError: Invalid mapping class names.
 >>> cm.relabel({0: "L1", 1: "L2", 2: "L2"})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmMatrixError: Mapping class names error
+pycm.pycm_obj.pycmMatrixError: Invalid mapping class names.
 >>> cp = Compare([cm, cm])
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmCompareError: The input type is supposed to be dictionary but it's not!
+pycm.pycm_obj.pycmCompareError: Input must be provided as a dictionary.
 >>> cp = Compare({"cm1": cm})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmCompareError: Lower than two confusion matrices is given for comparing. The minimum number of confusion matrix for comparing is 2.
+pycm.pycm_obj.pycmCompareError: At least 2 confusion matrices are required for comparison.
 >>> cp = Compare({"cm1": cm, "cm2": []})
 Traceback (most recent call last):
         ...
-pycm.pycm_obj.pycmCompareError: The input is supposed to consist of pycm.ConfusionMatrix object but it's not!
+pycm.pycm_obj.pycmCompareError: Input must be an instance of pycm.ConfusionMatrix.
```

Comment:
This error message is not accurate enough.
Input is a dictionary that consists of `pycm.ConfusionMatrix` objects not an instance of a `pycm.ConfusionMatrix`